pom.xml

```diff
@@ -8,7 +8,7 @@
     <name>DataStax Metrics Collector for Apache Cassandra</name>
 
     <properties>
-        <revision>0.2.0</revision>
+        <revision>0.3.3</revision>
         <dropwizard.version>1.0.2</dropwizard.version>
         <!-- <cassandra.version>4.0-beta4</cassandra.version> -->
         <cassandra.version>3.11.10</cassandra.version>
```

Comment:
Should this version be changed?

Reply:
It doesn't matter a lot because the build script will override it on release here:
https://github.com/datastax/metric-collector-for-apache-cassandra/blob/master/make_package.sh#L28
which is called with the tag version from here:
https://github.com/datastax/metric-collector-for-apache-cassandra/blob/master/.github/workflows/release.yaml#L18